### PR TITLE
Adding control flag for services

### PR DESCRIPTION
**Pull request by arunsanna**
### Description
This will allow to control the installation of service in the containers without systemd installed

### Issues Resolved

[List any existing issues this PR resolves]

### Check List

- [ ] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] All commits have been signed for the Developer Certificate of Origin. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD>

### CODE REVIEW

**Comment by tas50** [comment]
@arunsanna This will need the DCO signed off before it can be merged.

**Comment by damacus** [comment]
Hey @arunsanna,

this looks good. If you can add a changelog entry we're good to ship! 


Thanks,
Dan

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please include a <a href="https://github.com/sous-chefs/line-cookbook/blob/master/CHANGELOG.md">CHANGELOG</a> entry.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: attributes/default.rb [hunk 1]
```diff
@@ -110,3 +110,6 @@
 default['nginx']['ohai_plugin_enabled'] = true
 
 default['nginx']['load_modules'] = []
+
+#service flag controls intallation of the services in container which dont have systemd
```
Comment (josephholsten) [inline attributes/default.rb]: needs a space

**Comment by damacus** [comment]
## Closing due to inactivity. 

If this is still an issue please reopen or open another PR. 

Alternatively drop by the #sous-chefs channel on the [Chef Community Slack](http://community-slack.chef.io/) and we'll be happy to help! 

Thanks,
Sous-Chefs

**Pull request closed without merging**